scripts/nqp/actions/afflictions.py

```diff
@@ -49,10 +49,14 @@ def apply(self) -> Iterator[Tuple[EntityID, List[Effect]]]:
         """
         An iterator over pairs of (affected entity, [effects])
         """
+        entities = set()
```

Comment:
How does this work? Why are we making `entities` a `set`?

Reply:
This is just something auxiliary. It is checking we don't apply the afflictions multiple times to the same entity in the case it has multiple tiles. It's a set because it has a faster access time (O(1)) than a `List` for this case